proto/quicksilver/airdrop/v1/genesis.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package quicksilver.airdrop.v1;
+
+import "gogoproto/gogo.proto";
+import "quicksilver/airdrop/v1/params.proto";
+
+option go_package = "github.com/ingenuity-build/quicksilver/x/airdrop/types";
+
+// GenesisState defines the airdrop module's genesis state.
+message GenesisState {
+    Params params = 1 [ (gogoproto.nullable) = false ];
```

Comment:
This will need to contain the initial genesis airdrops - likely as a `repeated ClaimRecord`

Reply:
It will also need to be capable of exporting `repeated ZoneDrop`s